README.md

```diff
@@ -1,30 +1,29 @@
-# Glutil
+# glutil
```

Comment:
I would leave this capitalized. I don't see a reason for "glutil" to not to follow the rules for a proper noun except when specifically referring to the command line utility.

In my mind Glutil the project and `glutil` the command line application are separate things, `glutil` uses Glutil, and is provided by it, but the project itself doesn't need to be referred to as `glutil` with code formatting.

Reply:
Consistency is all I'm aiming for here 👍 

`glutil` for the command-line application and Python library and Glutil for the project, then?

Reply:
That was my intention when writing the readme